src/screens/WebsocketResearchScreen/SampleChatInfoScreen.tsx

```diff
@@ -247,9 +277,17 @@ const SampleChatInfoScreen = () => {
   const countParticipat = () => {
     return `(${channelInfo?.members?.length})`;
   };
+
+  const handleWidthChatName = () => {
+    if (params?.from === SIGNED) {
+      return styles.chatNameContainer;
+    }
+    return {};
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `handleWidthChatName` function has been introduced to conditionally return styles based on the `params.from` value. This is a good practice for dynamic styling.



<!-- This is an auto-generated comment by OSS CodeRabbit -->